dev-python/shtab/shtab-1.6.5.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11..12} pypy3 )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Automagic shell tab completion for Python CLI applications"
+HOMEPAGE="
+	https://github.com/iterative/shtab
+	https://pypi.org/project/shtab/
+"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="dev-python/setuptools-scm[${PYTHON_USEDEP}]"
+
+PATCHES=( "${FILESDIR}/${P}-pyproject.patch" )
```

Comment:
Did you submit the patch upstream?

Reply:
sent upstream. https://github.com/iterative/shtab/pull/162
@MocioF Can you add this link to the patch header?

Reply:
Add it as a comment inside the `PATCHES` array, please, so we can see it immediately when looking inside the ebuild file.